### PR TITLE
Create Remove-SPOListDesign.md

### PR DESCRIPTION
**Pull request by SaladiHarini**
Adding new cmdlets to support the Custom List Templates feature!

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Remove-SPOListDesign.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+Remove-SPOListDesign [-Identity] <SPOListDesignPipeBind> [<CommonParameters>]
+```
+
+## DESCRIPTION
+Removes a list design. It no longer appears in the UI for creating a new list. 
+
+## EXAMPLES
+
+### Example 1
+This example shows how to remove a list design.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Remove-SPOListDesign.md:25]:
```suggestion
```

File: sharepoint/sharepoint-ps/sharepoint-online/Remove-SPOListDesign.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+## EXAMPLES
+
+### Example 1
+This example shows how to remove a list design. 
+
+```powershell
+Remove-SPOListDesign 44252d09-62c4-4913-9eb0-a2a8b8d7f863
+```
+
+{{ Add example description here }}
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Remove-SPOListDesign.md:31]:
```suggestion
This example shows how to remove a list design. 
```

File: sharepoint/sharepoint-ps/sharepoint-online/Remove-SPOListDesign.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+Required: True
+Position: 0
+Default value: None
+Accept pipeline input: True (ByValue)
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Remove-SPOListDesign.md:51]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: sharepoint/sharepoint-ps/sharepoint-online/Remove-SPOListDesign.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+## INPUTS
+
+### Microsoft.Online.SharePoint.PowerShell.SPOListDesignPipeBind
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Remove-SPOListDesign.md:55]:
```suggestion
Microsoft.Online.SharePoint.PowerShell.SPOListDesignPipeBind
```

File: sharepoint/sharepoint-ps/sharepoint-online/Remove-SPOListDesign.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+## OUTPUTS
+
+### System.Object
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Remove-SPOListDesign.md:59]:
```suggestion
System.Object

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @SaladiHarini, kindly commit the suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
@SaladiHarini Please implement the suggested changes. Thanks

**Comment by yogkumgit** [comment]
@SaladiHarini Gentle reminder.

**Review by get-itips — CHANGES_REQUESTED** [review]
What about author metadata?

File: sharepoint/sharepoint-ps/sharepoint-online/Remove-SPOListDesign.md (hunk 5 above)
Comment (SaladiHarini) [reply sharepoint/sharepoint-ps/sharepoint-online/Remove-SPOListDesign.md:59]: I can't accept this change as this is making build fail. The output and input parameters are congruent to how platPS wants it, so removing the ### is making the whole PR fail.

Comment (SaladiHarini) [reply sharepoint/sharepoint-ps/sharepoint-online/Remove-SPOListDesign.md:59]: @yogkumgit / @JohanFreelancer9 ^

**Comment by get-itips** [comment]
Please see this article as an example on how to add author metadata 

https://raw.githubusercontent.com/MicrosoftDocs/OfficeDocs-SharePoint-PowerShell/main/sharepoint/sharepoint-ps/sharepoint-online/Get-SPOSite.md

**Comment by yogkumgit** [comment]
@SaladiHarini

**Comment by SaladiHarini** [comment]
> Please see this article as an example on how to add author metadata
> 
> https://raw.githubusercontent.com/MicrosoftDocs/OfficeDocs-SharePoint-PowerShell/main/sharepoint/sharepoint-ps/sharepoint-online/Get-SPOSite.md

@get-itips @yogkumgit I've added the author metadata, sorry I missed it in my previous PR as I was trying to find the cause for the build failing on me!

**Comment by get-itips** [comment]
Thank you, please also add ms.reviewer:

**Comment by SaladiHarini** [comment]
@tseward @yogkumgit @get-itips I've made the necessary suggested changes and also incorporated some feedback based on adjacent PRs, please take a look and let me know if this is good to go!

**Review by JohanFreelancer9 — APPROVED** [review]

File: sharepoint/sharepoint-ps/sharepoint-online/Remove-SPOListDesign.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
+---
+external help file: Microsoft.Online.SharePoint.PowerShell.dll-Help.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version:
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Remove-SPOListDesign.md:4]: Please complete online version metadata and add title metadata thanks

**Comment by yogkumgit** [comment]
@SaladiHarini

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tseward Please review. Thanks.

**Comment by yogkumgit** [comment]
@SaladiHarini please implement the suggested corrections. Thanks

**Comment by SaladiHarini** [comment]
@tseward I've made requested changes!

**Pull request merged**